src/app/modules/dashboard/container/auth-dashboard/auth-dashboard.component.html

```
@@ -0,0 +1 @@
+<p>auth-dashboard works!</p>
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)